sql/core/pom.xml

```diff
@@ -217,6 +217,35 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
is it the first time we shade something in SQL core? Can you refer to the PR that shade things in Spark core? Then it will be much easier to review this PR.

Reply:
OK. Refer to [this commit](https://github.com/apache/spark/commit/a15f6e31fc216d0d39bc1578e0da11b068b7821c) to shade Parquet now.